sql/system-database.md

```diff
@@ -0,0 +1,34 @@
+---
+title: The TiDB System Database
+category: user guide
+---
+
+The TiDB System Database is similar to MySQL, which contains tables that store information required by the server as it runs.
```

Comment:
as -> when. MySQL uses "as" but it is not a good usage because "as" has other meanings.